services/storage/object/controls.yaml

```diff
@@ -84,13 +84,17 @@ controls:
     test_requirements:
       - id: CCC.ObjStor.C03.TR01
         text: |
-          Object storage buckets cannot be deleted after creation.
+          When an object storage bucket is created then it must be protected from all deletion attempts, including both hard delete and soft delete, to ensure irrevocable data integrity and prevent any loss.
+        tlp_levels:
+          - tlp_red
```

Comment:
Are we able to confirm that this will be true for all TLP Red object storage instances?

Reply:
Removed